.vscode/settings.json

```diff
@@ -1,11 +1,32 @@
 {
   "[handlebars]": {
-    "editor.tabSize": 2
+    "editor.formatOnSave": false
```

Comment:
Prettier is not a good friend with Handlebars, so I am disabling the formatting on a save for this type. Also, I do enable auto formatting explicitly on file types, just to have more control over it.

Reply:
I usually keep these locally as user preferences, but I don't mind them on the repo, consistency.

Reply:
When I got onto a new machine I found it odd my usual setup wasn't working anymore... Obviously I forgot to move the config. But then I think having it on a repo makes more sense, everyone contributing would use the same...

Reply:
Assuming they use VS Code, but yea. I like them on the repo, it's cool.